tools/getkeys/index.js

```diff
@@ -131,39 +131,41 @@ class AzCliKeyVaultClient {
     }
 
     async getSecrets(vaultName) {
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-return
         return JSON.parse(await execAsync(`az keyvault secret list --vault-name ${vaultName}`));
     }
 
     async getSecret(vaultName, secretName) {
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-return
         return JSON.parse(await execAsync(`az keyvault secret show --vault-name ${vaultName} --name ${secretName}`));
     }
-};
+}
 
 class MsRestAzureKeyVaultClinet {
     static async get() {
-        const credentials = await msRestAzure.interactiveLogin();
+        const credentials = await interactiveLogin();
         return new MsRestAzureKeyVaultClinet(credentials);
     }
 
     constructor(credentials) {
-        this.client = new keyVault.KeyVaultClient(credentials);
+        this.client = new KeyVaultClient(credentials);
     }
 
     async getSecrets(vaultName) {
         return this.client.getSecrets(`https://${vaultName}.vault.azure.net/`);
     }
 
     async getSecret(vaultName, secretName) {
-        return this.client.getSecret(`https://${vaultName}.vault.azure.net/`, secretName, '');
+        return this.client.getSecret(`https://${vaultName}.vault.azure.net/`, secretName, "");
     }
 }
 
 async function getClient() {
-    return await AzCliKeyVaultClient.get();
+    return AzCliKeyVaultClient.get();
 
     // Disabling fallback to REST client while we decide how to streamline the getkeys tool
 
-    // const primary = await AzCliKeyVaultClient.get();
+    // const primary = AzCliKeyVaultClient.get();
```

Comment:
We actually need to keep the `await` in this case. If we at some point uncomment this code, the next line does require that this line is awaited.

Reply:
Change back then